By Salwa Salsabila | Universitas Dian Nuswantoro

# Arithmetic and Variables

Make calculations, and define and modify variables.

## Printing

Di Python, kita dapat memerintahkan komputer untuk menampilkan pesan dengan menggunakan fungsi *print()* dan memberikan pesan yang diinginkan di dalam tanda kurung, diapit tanda kutip. Misalnya, jika kita ingin mencetak pesan "Helo, world!", kita menulis kode berikut:

In [1]:
print("Hello, world!")

Hello, world!


Dengan menggunakan perintah print, kita memberi tahu komputer untuk mencetak atau menampilkan teks yang ada di dalam tanda kurung ganda (") ke layar. Dalam hal ini, teks yang akan dicetak adalah "Hello, world!", yang merupakan pesan yang sering digunakan dalam tutorial pemrograman sebagai langkah pertama dalam mempelajari sebuah bahasa pemrograman baru.

Jadi, ketika kita menjalankan perintah print("Hello, world!"), kita akan melihat pesan "Hello, world!" muncul di layar komputer sebagai hasil dari program yang kita tulis.

## Arithmetic

Kita bisa juga menampilkan hasil dari beberapa operasi matematika (seperti penambahan, pengurangan, perkalian, atau pembagian).

In [2]:
print(1 + 2)

3


Perintah *print(1 + 2)* dalam bahasa pemrograman Python adalah cara untuk menampilkan hasil penjumlahan dari angka 1 dan 2 ke layar komputer. Dalam kasus ini, tanda kurung digunakan untuk menunjukkan bahwa kita ingin mencetak hasil operasi matematika yang terdapat di dalamnya.

Python akan menjumlahkan angka 1 dan 2, dan output yang dihasilkan 3. Jadi, hasil dari perintah ini adalah angka 3. Ini adalah contoh sederhana dari bagaimana kita dapat menggunakan Python untuk melakukan operasi matematika dasar dan menampilkan hasilnya kepada pengguna.

In [3]:
print(9 - 5)

4


Perintah *print(9 - 5)* dalam bahasa pemrograman Python adalah cara untuk menampilkan hasil pengurangan dari angka 9 dan 5 ke layar komputer. Ketika kode ini dieksekusi, Python akan mengurangkan angka 5 dari angka 9, yaitu 4, dan kemudian mencetak hasilnya ke layar. Jadi, hasil dari perintah ini adalah angka 4

In [4]:
print(((1 + 3) * (9 - 2) / 2) ** 2)

196.0


Perintah *print(((1 + 3) * (9 - 2) / 2) ** 2)* dalam bahasa pemrograman Python adalah cara untuk menghitung hasil ekspresi matematika yang kompleks dan menampilkan hasilnya ke layar komputer. Dalam ekspresi ini, terdapat beberapa operasi matematika yang dilakukan secara berurutan.

1. (1 + 3): Operasi penjumlahan dari angka 1 dan 3, menghasilkan 4.
2. (9 - 2): Operasi pengurangan dari angka 9 dan 2, menghasilkan 7.
3. (4 * 7): Operasi perkalian dari hasil penjumlahan dan pengurangan sebelumnya, menghasilkan 28.
4. (28 / 2): Operasi pembagian hasil perkalian sebelumnya dengan angka 2, menghasilkan 14.
5. ** 2: Operasi pemangkatan hasil pembagian sebelumnya ke pangkat 2, menghasilkan 196.

Hasil akhir dari ekspresi ini adalah angka 196.

## Comments

Komentar adalah bagian dari kode yang tidak dieksekusi oleh Python. Mereka digunakan untuk memberikan penjelasan atau dokumentasi tentang apa yang dilakukan oleh baris kode yang terkait.

Contohnya, di sel kode berikut, kita mengalikan 3 dengan 2. Kita tambahkan komentar (# Multiply 3 by 2) di atas kode untuk menjelaskan apa yang dilakukan oleh kode tersebut.

In [5]:
# Multiply 3 by 2
print(3 * 2)

6


Untuk menandai bahwa suatu baris adalah komentar dalam Python, gunakan tanda pagar (#) sebagai karakter pertama.

Setelah menemukan tanda pagar, Python akan menganggap baris tersebut sebagai komentar dan akan mengabaikannya sepenuhnya. Ini penting karena Python mengikuti aturan yang sangat ketat, dan hanya akan menghasilkan kesalahan jika tidak dapat memahami kode yang diberikan.

Contohnya terlihat pada sel kode di bawah ini. Python akan menghasilkan kesalahan jika tanda pagar dihilangkan karena teks dalam komentar bukanlah kode Python yang valid dan tidak dapat diinterpretasikan dengan benar.

In [7]:
Multiply 3 by 2

SyntaxError: invalid syntax (3750420471.py, line 1)

## Variables